DesktopModules/Vanjaro/UXManager/Extensions/Menu/EmailServiceProvider/Controllers/SettingController.cs

```diff
@@ -24,15 +24,16 @@ internal static List<IUIData> GetData(PortalSettings PortalSettings)
         {
             bool IsSuperUser = UserController.Instance.GetCurrentUserInfo().IsSuperUser;
             string mode = IsSuperUser ? "h" : "p";
+            HostController hostController = new HostController();
             Dictionary<string, IUIData> Settings = new Dictionary<string, IUIData>
             {
                 { "IsSuperUser", new UIData { Name = "IsSuperUser", Options = IsSuperUser } },
                 { "SMTPmode", new UIData { Name = "SMTPmode", Options = IsSuperUser? PortalController.GetPortalSetting("SMTPmode", PortalSettings.PortalId, mode) == mode:false } },
-                { "Host_Server", new UIData { Name = "Host_Server", Value = HostController.Instance.GetString("SMTPServer") } },
-                { "Host_Username", new UIData { Name = "Host_Username", Value = HostController.Instance.GetString("SMTPUsername") } },
-                { "Host_Password", new UIData { Name = "Host_Password", Value = HostController.Instance.GetEncryptedString("SMTPPassword", Config.GetDecryptionkey()) } },
-                { "Host_Email", new UIData { Name = "Host_Email", Value = HostController.Instance.GetString("SMTPEmail",PortalSettings.Email) } },
-                { "Host_EnableSSL", new UIData { Name = "Host_EnableSSL", Options = HostController.Instance.GetBoolean("SMTPEnableSSL", false) } },
+                { "Host_Server", new UIData { Name = "Host_Server", Value = hostController.GetString("SMTPServer") } },
+                { "Host_Username", new UIData { Name = "Host_Username", Value = hostController.GetString("SMTPUsername") } },
+                { "Host_Password", new UIData { Name = "Host_Password", Value = hostController.GetEncryptedString("SMTPPassword", Config.GetDecryptionkey()) } },
```

Comment:
Can we create SettingManager.GetHostSetting() similar to GetHostSetting()?

Reply:
changes done